spec/lib/govuk_component/link_helpers_spec.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+RSpec.describe GovukComponent::LinkHelpers, type: :helper do
+  include RSpecHtmlMatchers
+
+  describe '#govuk_back_link_to' do
+    subject(:markup) { helper.govuk_back_link_to(*args) }
+
+    context 'default component' do
+      let(:args) { ['GovUK', 'https://www.gov.uk'] }
+
+      it 'adds link with govuk class' do
+        is_expected.to have_tag(:a, with: { class: 'govuk-back-link', href: 'https://www.gov.uk' }, text: 'GovUK')
+      end
+    end
+
+    context 'with a custom class' do
+      let(:args) { ['GovUK', 'https://www.gov.uk', class: 'my-custom-class1 my-custom-class2'] }
+
+      it 'adds email link with custom classes, prepended by govuk class' do
```

Comment:
Should this be 'adds link with custom classes...' rather than 'adds email link with custom classes'?